modules/client/src/connext.ts

```diff
@@ -599,14 +599,14 @@ export class ConnextInternal extends ConnextChannel {
               `More than ${maxBlocks} have passed, blocks elapsed: ${blockNumber - startingBlock}`,
             );
           }
-        });
+        } catch (e) {
+          if (e.includes(`More than ${maxBlocks} have passed`)) {
```

Comment:
Still confused by this try-catch. What is actually being caught here? Why would the error include the string "More than ${maxBlocks} have passed"? Looking at the `checkForUserWithdrawal` function, I don't see it throwing an error with that message.

Reply:
the promise i create throws that error if more than 15 blocks pass, then pings the hub to resubmit